schemas/swagger.yaml

```diff
@@ -96,7 +96,9 @@ paths:
                     removedDefinitions:
                       type: boolean
                 patches:
-                  $ref: /schemas/curation-1.0.json
+                  type: array
+                  items:
+                    $ref: /schemas/curation-1.0.json
```

Comment:
Actually, I believe this is still not correct. If I understand the test code correctly, `patches` is not an array of `curation`, but an array of a `patch` object which in turn contains `coordinates` and `revisions`, and `revisions` is a map of versions to `curation`.

Reply:
Nice catch again. An 's' was missing.